src/tutorial-panel/components/tutorial-detail-view/task.tsx

```diff
@@ -58,7 +58,7 @@ export function Task({ task, taskIndex, currentTaskIndex, expanded, onToggleExpa
 
         <div className={styles['expandable-section-wrapper']}>
           <InternalExpandableSection
-            headerText={
+            header={
```

Comment:
Do we plan to update this in some future?

Reply:
I think it would make sense, but the current reality is that having non-standard-sized headings in expandable section isn't really supported by the component, so might need some additions there to support this in a more elegant way.